OpenDreamRuntime/Procs/ProcDecoder.cs

```diff
@@ -142,6 +142,8 @@ public ITuple DecodeInstruction() {
             case DreamProcOpcode.Gradient:
             case DreamProcOpcode.Rgb:
                 return (opcode, (DMCallArgumentsType)ReadByte(), ReadInt());
+            case DreamProcOpcode.Animate:
```

Comment:
The bodies are the same so you can put the two cases together instead.
```suggestion
            case DreamProcOpcode.Rgb:
            case DreamProcOpcode.Animate:
```